Wikipedia/Code/PageIssuesTableViewController.swift

```diff
@@ -14,7 +14,9 @@ class PageIssuesTableViewController: UITableViewController {
         
         self.tableView.estimatedRowHeight = 90.0
         self.tableView.rowHeight = UITableView.automaticDimension
-        self.tableView.separatorStyle = UITableViewCell.SeparatorStyle.none
+        self.tableView.separatorStyle = UITableViewCell.SeparatorStyle.singleLine // changed from .none to .singleLine
+        self.tableView.separatorColor = UIColor.red //added this line to change color of seperator
```

Comment:
Nice work! 

The code itself is descriptive enough, so you can remove these comments.